sources/packages/api/src-sql/StudentScholasticStandings/Revert-scholastic-standing-approval-fields.sql

```diff
@@ -0,0 +1,24 @@
+-- Revert scholastic standing approval fields.
```

Comment:
As a suggestion, I would recommend to rename the file as Revert-remove-scholastic-standing-approval-fields.sql to help understand adding columns. Otherwise migration.ts is the place that we need to check for the relevance.